addon/components/ember-flatpickr.js

```diff
@@ -1,10 +1,73 @@
-import Component from '@ember/component';
+/** @documenter yuidoc */
+
 import Flatpickr from 'ember-flatpickr/mixins/flatpickr';
+import TextField from '@ember/component/text-field';
```

Comment:
Instead of extending `Component` extend `TextField` and get all of it's benefits.